exp/docgen/www/index.php

```diff
@@ -6,17 +6,35 @@
     
     $Path = isset( $_SERVER[ 'QUERY_STRING' ] ) ? trim( $_SERVER[ 'QUERY_STRING' ], '/' ) : '';
     $RenderLayout = !isset( $_SERVER[ 'HTTP_X_PJAX' ] ) || $_SERVER[ 'HTTP_X_PJAX' ] !== 'true';
-    
-    if (substr($Path, 0, 8) === '__search')
-        $RenderLayout = false;
-    
+
+    // Stupid bugfix when clicking on homepage
+    if (substr($Path, 0, 6) === '_pjax=') {
```

Comment:
Basically, mod_rewrite doesn't pass in path correctly when you try to load homepage using pjax (`/new-api/?_pjax=testest`) resulting in a 404 error. There's most likely a cleaner way of fixing this via `.htaccess`, but I couldn't figure it out.